examples/client/client.c

```diff
@@ -3558,6 +3558,21 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
         err_sys("unable to get SSL object");
     }
 
+#ifdef WOLFSSL_DUAL_ALG_CERTS
+    /* Set preference for verfication of both the native and alternative chains.
+     */
+    byte cks_order[3] = {
```

Comment:
This will cause mixed declaration. Needs to be at top of function. Can / should this be `const`?

Reply:
Prefer to keep it close together.  Compromise, open a new scope here.